torch/csrc/jit/serialization/export.cpp

```diff
@@ -1370,10 +1370,26 @@ void check_onnx_proto(const std::string& proto_string, bool full_check) {
     throw std::runtime_error("Invalid ONNX proto string.");
     return;
   }
+  // baseline check
   onnx::checker::check_model(model);
-
+  onnx::shape_inference::InferShapes(model);
+  // full check including strict shape inference check
+  // Note: onnx::checker::check_model doesn't
+  // include non-strict model onnx::shape_inference::InferShapes(model).
+  // That's why we need to write another onnx::checker::check_model(model,
+  // full_check) here.
```

Comment:
Just checking - was this reworded?

Reply:
I think it's Notice that -> Note:

Reply:
reword: 
1. clarify non-strict mode
2. call check_model twice reason